gradle.properties

```diff
@@ -10,7 +10,7 @@ sizeofVersion = 0.3.0
 jaxbVersion = 2.3.1
 
 # Terracotta clustered
-terracottaPlatformVersion = 5.8.0-pre23
+terracottaPlatformVersion = 5.8.0-pre25
```

Comment:
@GaryWKeim @chrisdennis : FYI updated to -25 directly (release in progress) to also include Gary's changed for M&M